python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -619,6 +650,19 @@ def terminate_scheduled_nodes(self):
         # explicitly disabled.
         if self.worker_rpc_drain:
             self.drain_nodes_via_gcs(self.nodes_to_terminate)
+        # Clean up multi-host replicas to delete
+        for node in self.nodes_to_terminate:
+            tags = self.provider.node_tags(node)
+            if TAG_RAY_REPLICA_INDEX in tags:
+                replica_index = tags[TAG_RAY_REPLICA_INDEX]
+                if replica_index is not None:
```

Comment:
When will `replica_index` be None?

Reply:
Done in 47e46ac, it will never be None if returned by node_tags now.